packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -485,115 +551,175 @@ export default defineComponent({
-        ...[
-          {
-            name: 'name',
-            title: this.$gettext('Name'),
-            type: 'slot',
-            width: 'expand',
-            wrap: 'truncate'
-          },
-          {
-            name: 'indicators',
-            title: this.$gettext('Shares'),
-            type: 'slot',
-            alignH: 'right',
-            wrap: 'nowrap',
-            width: 'shrink'
-          },
-          {
-            name: 'size',
-            title: this.$gettext('Size'),
-            type: 'slot',
-            alignH: 'right',
-            wrap: 'nowrap',
-            width: 'shrink'
-          },
-          {
-            name: 'status',
-            title: this.$gettext('Status'),
-            type: 'slot',
-            alignH: 'right',
-            wrap: 'nowrap',
-            width: 'shrink'
-          },
-          this.hasTags
-            ? {
-                name: 'tags',
-                title: this.$gettext('Tags'),
-                type: 'slot',
-                alignH: 'right',
-                wrap: 'nowrap',
-                width: 'shrink'
-              }
-            : {},
-          {
-            name: 'owner',
-            title: this.$gettext('Shared by'),
-            type: 'slot',
-            alignH: 'right',
-            wrap: 'nowrap',
-            width: 'shrink'
-          },
-          {
-            name: 'sharedWith',
-            title: this.$gettext('Shared with'),
-            type: 'slot',
-            alignH: 'right',
-            wrap: 'nowrap',
-            width: 'shrink'
-          },
-          {
-            name: 'mdate',
-            title: this.$gettext('Modified'),
-            type: 'slot',
-            alignH: 'right',
-            wrap: 'nowrap',
-            width: 'shrink',
-            accessibleLabelCallback: (item) =>
-              this.formatDateRelative(item.mdate) + ' (' + this.formatDate(item.mdate) + ')'
-          },
-          {
-            name: 'sdate',
-            title: this.$gettext('Shared on'),
-            type: 'slot',
-            alignH: 'right',
-            wrap: 'nowrap',
-            width: 'shrink',
-            accessibleLabelCallback: (item) =>
-              this.formatDateRelative(item.sdate) + ' (' + this.formatDate(item.sdate) + ')'
-          },
-          {
-            name: 'ddate',
-            title: this.$gettext('Deleted'),
-            type: 'slot',
-            alignH: 'right',
-            wrap: 'nowrap',
-            width: 'shrink',
-            accessibleLabelCallback: (item) =>
-              this.formatDateRelative(item.ddate) + ' (' + this.formatDate(item.ddate) + ')'
-          }
-        ]
-          .filter((field) => {
-            const hasField = Object.prototype.hasOwnProperty.call(firstResource, field.name)
-            if (!this.fieldsDisplayed) {
-              return hasField
+      if (firstResource?.type === 'space') {
+        fields.push(
+          ...[
+            {
+              name: 'icon',
```

Comment:
one other issue with this: we want to see the actual space thumbnails in the list. the icon is not sufficient.